README.md

```diff
@@ -56,6 +56,37 @@ The bot can be configured by adding a `.mention-bot` file to the base directory
 
 The glob matching is an extended form of glob syntax performed by [`minimatch`](https://github.com/isaacs/minimatch), with the default options; read [the `minimatch` README](https://github.com/isaacs/minimatch/blob/master/README.md) for more details.
 
+The following default config:
```

Comment:
Can you kill this whole block, we already have it in the README above. I want to make sure when someone adds a new option we don't need to edit yet another line of code :)